authors/apps/authentication/models.py

```diff
@@ -117,14 +117,14 @@ def get_short_name(self):
         return self.username
 
     @staticmethod
-    def encode_auth_token(username):
+    def encode_auth_token(email):
         """Generates auth token."""
         try:
             payload = {
                 'exp': datetime.datetime.utcnow() + datetime.timedelta(
                     days=2, seconds=20000),
```

Comment:
@engjames please reduce the token expiration time to atleast 15 minutes for development and testing purposes.